src/ert/gui/tools/manage_experiments/manage_experiments_panel.py

```diff
@@ -142,3 +197,69 @@ def eventFilter(self, a0: Optional[QObject], a1: Optional[QEvent]) -> bool:
         if a1 is not None and a1.type() == QEvent.Type.Close:
             self.notifier.emitErtChange()
         return super().eventFilter(a0, a1)
+
+    def _add_initialize_from_design_matrix_tab(self) -> None:
```

Comment:
We were talking about doing this with `Ensemble experiment` instead, did you check how much work that would take?